src/gui/DatabaseWidget.h

```diff
@@ -309,6 +310,9 @@ private slots:
     // Autoreload
     bool m_blockAutoSave;
 
+    // Autosave delay
+    QPointer<QTimer> m_autosaveDelayTimer;
```

Comment:
I would name as `m_autosaveTimer`.

Reply:
Fixed in [3a1e023](https://github.com/keepassxreboot/keepassxc/pull/9100/commits/3a1e0231d88af17d161555fa7182d6b9166b162a)